src/repository/migrations/0018_auto_20201027_1110.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-27 11:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('repository', '0017_auto_20200930_1629'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='preprintversion',
+            name='abstract',
+            field=models.TextField(blank=True, help_text='Please avoid pasting content from word processors as they can add unwanted styling to the abstract. You can retype the abstract here or copy and paste it into notepad/a plain text editor before pasting here.', null=True),
+        ),
+        migrations.AddField(
+            model_name='preprintversion',
+            name='title',
+            field=models.CharField(default='', help_text='Your article title', max_length=300),
+            preserve_default=False,
+        ),
```

Comment:
I think we need a migration to back-populate all existing `PreprintVersion` and `VersionQueue`.

Alternatively assume an empty field as "unchanged" when we render the history (in which case we should handle that in `VersionQueue.approve` too). This approach is a bit more implicit application-wise but does save us from having dobule copies of abstract and title for each version.